packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -730,7 +730,7 @@ export class DeltaManager
 
         if (this.readonly === true) {
             assert(this.readOnlyInfo.readonly === true, 0x1f0 /* "Unexpected mismatch in readonly" */);
-            const error = CreateContainerError("Op is sent in read-only document state", {
+            const error = new GenericError("Op is sent in read-only document state", undefined /* error */, {
```

Comment:
```suggestion
            const error = new GenericError("deltaMangerReadonlySubmit", undefined /* error */, {
```
maybe user error?

Reply:
> maybe user error?

I'm not deeply familiar with who all calls `submit` and when, but in any case it's several layers away from public API's, so I'm inclined to leave it, since it could be our code missing a readonly check in some cases?  Feel free to push back if you think it over and do think it's appropriate to call it UsageError at this layer.